### PR TITLE
Review geth/jail README.md

### LINKED ISSUE #625: Markdown problems on Jail README
# Problem

Current [Jail package README](https://github.com/status-im/status-go/blob/develop/geth/jail/README.md) presents some problems rendering on Github.

# Implementation

Fix Markdown problems in order to have a proper README

# Acceptance Criteria

README should render properly on github

### PR DESCRIPTION
**Pull request by adriacidre**
Fixed minor problems on Jail README markdown

Closes #625

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]
Thanks! LGTM

File: geth/jail/README.md [hunk 1]
```diff
@@ -29,73 +29,83 @@ Otto virtual machine and lives forever, but that may change in the future.
...
 
 In order to capture response one may use following approach:
 
-```
+```go
 err = cell.Set("__captureResponse", func(val string) otto.Value {
-	fmt.Println("Captured response from callback:", val)
-	return otto.UndefinedValue()
+
+```
```
Comment (divan) [inline geth/jail/README.md]: Somehow this has rendered improperly. Have you used `autoreadme` tool to convert from doc.go to README?

**Review by divan — CHANGES_REQUESTED** [review]

File: geth/jail/README.md (hunk 1 above)
Comment (adriacidre) [reply geth/jail/README.md]: Hey @divan good catch. Not sure what happened here maybe I ran auroreadme without the `-template` flag.

May be a good idea to have a Makefile or even a git hook so any change on the go.doc file automatically generates the README.

Anyway that's fixed now

Comment (divan) [reply geth/jail/README.md]: I'm afraid it's the problem with upstream `autoreadme`. I used this patched one: https://github.com/divan/autoreadme. Could you please try it?

Comment (adriacidre) [reply geth/jail/README.md]: With your fork:

Running `autoreadme -f -template=doc.go` produces no changes.

On the other hand simply running `autoreadme -f` produces an strange result with problems on some code blocks:

screen shot 2018-02-07 at 14 07 58

File: geth/jail/doc.go [hunk 2]
```diff
@@ -1,86 +1,91 @@
...
-  |ChatID 1 | |ChatID 2 | |ChatID 3 |  |ChatID N |
-  |+-------+| |+-------+| |+-------+|  |+-------+|
-  ||Otto VM|| ||Otto VM|| ||Otto VM||  ||Otto VM||
-  |+-------+| |+-------+| |+-------+|  |+-------+|
-  || Loop  || || Loop  || || Loop  ||  || Loop  ||
-  ++-------++ ++-------++ ++-------++  ++-------++
-
-Cells
-
+```
```
Comment (divan) [inline geth/jail/doc.go]: So here is the problem: godoc is not in Markdown. It uses indentation for code blocks, not the ``` clauses.

**Comment by adriacidre** [comment]
Damn it, didn't noticed about the indentation.

I've changed all markdown code blocks by indentation instead. I think final [README](https://github.com/adriacidre/status-go/blob/33ea1dd07755e6863ad3aaa511d52ce2a4a44c11/geth/jail/README.md) looks correct now.

**Review by b00ris — APPROVED** [review]
Thanks! LGTM

**Review by divan — APPROVED** [review]
LGTM

**Pull request merged**